index.js

```diff
@@ -1,74 +1,89 @@
+  static getFromTheStorage() {
+    const storage = localStorage.getItem('books') === null
+      ? [] : JSON.parse(localStorage.getItem('books'));
+    return storage;
+  }
 }
+let books = Storage.getFromTheStorage();
 
-function addBook(title, author) {
-  const book = {
-    id: Date.now(),
-    title,
-    author,
-  };
-  books.push(book);
-  addElementsTopage(books);
-  addToLocalStorage(books);
-}
+let count = 1;
+class Libraray {
+  static displayData() {
+    booksDiv.innerHTML = '';
+    books.forEach((book) => {
+      const div = document.createElement('div');
+      div.className = 'book';
+      if (count % 2 === 0) {
+        div.classList.add('gray');
+      }
+      count += 1;
+      div.setAttribute('data-id', book.id);
+      const text = document.createElement('p');
+      text.append(document.createTextNode(book.title));
+      text.append(document.createTextNode(' by '));
+      text.append(document.createTextNode(book.author));
+      div.append(text);
+      const remove = document.createElement('button');
+      remove.className = 'remove';
+      remove.appendChild(document.createTextNode('Remove'));
+      div.appendChild(remove);
+      booksDiv.append(div);
+    });
+  }
 
-function getDataFromLocalStorage() {
-  const data = localStorage.getItem('books');
-  if (data) {
-    const books = JSON.parse(data);
-    addElementsTopage(books);
+  static clearInput() {
+    title.value = '';
+    author.value = '';
   }
-}
 
-getDataFromLocalStorage();
+  static removebook() {
+    booksDiv.addEventListener('click', (e) => {
+      if (e.target.classList.contains('remove')) {
+        e.target.parentElement.remove();
+      }
+      const btnId = e.target.parentElement.getAttribute('data-id');
+      Libraray.deleteBook(btnId);
+    });
+  }
 
-add.onclick = function () {
-  if (tiltle.value !== '' && author.value !== '') {
-    addBook(tiltle.value, author.value);
-    tiltle.value = '';
-    author.value = '';
+  static deleteBook(bookID) {
+    books = books.filter((book) => book.id !== +bookID);
+    Storage.addToLocalStorage(books);
   }
-};
-function deleteBook(bookID) {
-  books = books.filter((book) => book.id != bookID);
-  addToLocalStorage(books);
 }
-booksDiv.addEventListener('click', (e) => {
-  if (e.target.classList.contains('remove')) {
-    deleteBook(e.target.parentElement.getAttribute('data-id'));
-    e.target.parentElement.remove();
+add.addEventListener('click', () => {
+  if (author.value !== '' && title.value !== '') {
+    const id = Date.now();
+
+    const newBook = new Book(id, title.value, author.value);
+    books.push(newBook);
+    Libraray.displayData();
+    Libraray.clearInput();
+    Storage.addToLocalStorage(books);
   }
 });
-/* eslint eqeqeq: "off" */
\ No newline at end of file
+window.addEventListener('DOMContentLoaded', () => {
+  Libraray.displayData();
+  // remove from the dom
+  Libraray.removebook();
+});
+/* eslint max-classes-per-file: "off" */
```

Comment:
- [ ] Let's try not to disable linters as they help us write cleaner code. You can consider splitting the code in this file into different files